### PR TITLE
Pre-wrap Lines by default, setting to change to overflow

### PR DESCRIPTION
**Pull request by NWuensche**
Fix one of the requests in #734 .

### CODE REVIEW

**Comment by NWuensche** [comment]
Of course, I will squash everything again once this PR is approved.

**Comment by di72nn** [comment]
Maybe also make it clear that the option is for preformatted text (code blocks, etc.), rather then any long lines.

**Comment by NWuensche** [comment]
I hope everything is ready now.

File: app/src/main/res/xml/preferences.xml [hunk 1]
```diff
@@ -104,6 +104,14 @@
                 android:title="@string/pref_name_ui_article_textAlignment_justify"
                 android:summary="@string/pref_desc_ui_article_textAlignment_justify"
                 android:defaultValue="true"/>
+            <ListPreference
+                android:key="@string/pref_key_ui_article_handlePreformattedText"
+                android:title="@string/pref_name_ui_article_handlePreformattedText"
+                android:summary="@string/pref_desc_ui_article_handlePreformattedText"
+                android:dialogTitle="@string/pref_name_ui_article_handlePreformattedText"
+                android:entries="@array/pref_option_handlePreformattedText_values"
+                android:entryValues="@array/pref_option_handlePreformattedText_values"
```
Comment (di72nn) [inline app/src/main/res/xml/preferences.xml]: `android:entries` is for user-facing localized values and `android:entryValues` is for internal values. See `pref_option_autoSync` for example.

Comment (NWuensche) [reply app/src/main/res/xml/preferences.xml]: Should be changed now. I used the entries for the CSS-classes, I hope this is correct.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -204,6 +204,12 @@
     <string name="pref_desc_ui_article_fontSerif">Serif font for articles</string>
     <string name="pref_name_ui_article_textAlignment_justify">Text alignment: Justify</string>
     <string name="pref_desc_ui_article_textAlignment_justify">Stretches lines to equal width (like in newspapers)</string>
+    <string name="pref_name_ui_article_handlePreformattedText">Handling Long Preformatted Text</string>
+    <string name="pref_desc_ui_article_handlePreformattedText">How To handle preformatted text that is too long to show completely.</string>
+    <string-array name="pref_option_handlePreformattedText" translatable="true">
```
Comment (di72nn) [inline app/src/main/res/values/strings.xml]: `translatable="true"` is not needed here and in `values-de`.
Otherwise looks ok.

Comment (NWuensche) [reply app/src/main/res/values/strings.xml]: I did the rebase and also applied your change.

**Pull request merged**